spotinst/resource_spotinst_ocean_right_sizing_rule.go

```diff
@@ -315,10 +320,18 @@ func expandAttachWorkloadsConfig(data interface{},
 	return spec, nil
 }
 
-func expandNamespaces(data interface{}) ([]*right_sizing.Namespace, error) {
+func expandNamespaces(data interface{}, ruleName string, oceanId *string, meta interface{}) ([]*right_sizing.Namespace, error) {
 	list := data.(*schema.Set).List()
 	namespaces := make([]*right_sizing.Namespace, 0, len(list))
 
+	input := &right_sizing.ReadRightsizingRuleAttachedWorkloadsInput{
+		RuleName: spotinst.String(ruleName),
+		OceanId:  oceanId,
+	}
+	resp, err := meta.(*Client).ocean.RightSizing().ReadRightsizingRuleAttachedWorkloads(context.Background(), input)
+	log.Print(resp)
```

Comment:
please remove print statements which are not necessary

Reply:
removed print statements